### ISSUE TITLE
cattrs TypeError

### ISSUE THREAD

**Issue by JackieZhai** [scalableminds/webknossos#8040]
## Context

After `pip install "webknossos[all]"`, start CLI or just `import webknossos`.

## Expected Behavior

No Errors.

## Current Behavior

The package `cattrs` raises `TypeError`.
```
Traceback (most recent call last):
  File "/Users/haozhai/miniconda3/envs/WK/bin/webknossos", line 5, in <module>
    from webknossos.cli.main import app
  File "/Users/haozhai/miniconda3/envs/WK/lib/python3.9/site-packages/webknossos/__init__.py", line 27, in <module>
    from .administration import *
  File "/Users/haozhai/miniconda3/envs/WK/lib/python3.9/site-packages/webknossos/administration/__init__.py", line 2, in <module>
    from .project import Project
  File "/Users/haozhai/miniconda3/envs/WK/lib/python3.9/site-packages/webknossos/administration/project.py", line 6, in <module>
    from ..client.api_client.models import ApiProject
  File "/Users/haozhai/miniconda3/envs/WK/lib/python3.9/site-packages/webknossos/client/__init__.py", line 2, in <module>
    from ._upload_dataset import LayerToLink
  File "/Users/haozhai/miniconda3/envs/WK/lib/python3.9/site-packages/webknossos/client/_upload_dataset.py", line 12, in <module>
    from ..dataset import Dataset, Layer, RemoteDataset
  File "/Users/haozhai/miniconda3/envs/WK/lib/python3.9/site-packages/webknossos/dataset/__init__.py", line 14, in <module>
    from .dataset import Dataset, RemoteDataset
  File "/Users/haozhai/miniconda3/envs/WK/lib/python3.9/site-packages/webknossos/dataset/dataset.py", line 43, in <module>
    from ._utils import pims_images
  File "/Users/haozhai/miniconda3/envs/WK/lib/python3.9/site-packages/webknossos/dataset/_utils/pims_images.py", line 31, in <module>
    from ..mag_view import MagView
  File "/Users/haozhai/miniconda3/envs/WK/lib/python3.9/site-packages/webknossos/dataset/mag_view.py", line 25, in <module>
    from .properties import MagViewProperties
  File "/Users/haozhai/miniconda3/envs/WK/lib/python3.9/site-packages/webknossos/dataset/properties.py", line 330, in <module>
    make_dict_structure_fn(
  File "/Users/haozhai/miniconda3/envs/WK/lib/python3.9/site-packages/cattrs/gen/__init__.py", line 772, in make_dict_structure_fn
    return make_dict_structure_fn_from_attrs(
  File "/Users/haozhai/miniconda3/envs/WK/lib/python3.9/site-packages/cattrs/gen/__init__.py", line 406, in make_dict_structure_fn_from_attrs
    handler = find_structure_handler(
  File "/Users/haozhai/miniconda3/envs/WK/lib/python3.9/site-packages/cattrs/gen/_shared.py", line 52, in find_structure_handler
    handler = c.get_structure_hook(type, cache_result=False)
  File "/Users/haozhai/miniconda3/envs/WK/lib/python3.9/site-packages/cattrs/converters.py", line 562, in get_structure_hook
    else self._structure_func.dispatch_without_caching(type)
  File "/Users/haozhai/miniconda3/envs/WK/lib/python3.9/site-packages/cattrs/dispatch.py", line 134, in dispatch_without_caching
    res = self._function_dispatch.dispatch(typ)
  File "/Users/haozhai/miniconda3/envs/WK/lib/python3.9/site-packages/cattrs/dispatch.py", line 75, in dispatch
    return handler(typ, self._converter)
  File "/Users/haozhai/miniconda3/envs/WK/lib/python3.9/site-packages/cattrs/cols.py", line 118, in list_structure_factory
    handler = converter.get_structure_hook(elem_type)
  File "/Users/haozhai/miniconda3/envs/WK/lib/python3.9/site-packages/cattrs/converters.py", line 560, in get_structure_hook
    self._structure_func.dispatch(type)
  File "/Users/haozhai/miniconda3/envs/WK/lib/python3.9/site-packages/cattrs/dispatch.py", line 134, in dispatch_without_caching
    res = self._function_dispatch.dispatch(typ)
  File "/Users/haozhai/miniconda3/envs/WK/lib/python3.9/site-packages/cattrs/dispatch.py", line 76, in dispatch
    return handler(typ)
  File "/Users/haozhai/miniconda3/envs/WK/lib/python3.9/site-packages/cattrs/converters.py", line 636, in _gen_attrs_union_structure
    dis_fn = self._get_dis_func(cl, use_literals=use_literals)
  File "/Users/haozhai/miniconda3/envs/WK/lib/python3.9/site-packages/cattrs/converters.py", line 962, in _get_dis_func
    return create_default_dis_func(
  File "/Users/haozhai/miniconda3/envs/WK/lib/python3.9/site-packages/cattrs/disambiguators.py", line 159, in create_default_dis_func
    raise TypeError(f"{cl} has no usable non-default attributes")
TypeError: <class 'webknossos.dataset.properties.LayerProperties'> has no usable non-default attributes
```

## Steps to Reproduce the bug

1. pip install "webknossos[all]"
2. import webknossos

## Your Environment for bug

- Operating System and version: both MacOS 14.6.1 and Ubuntu 18.04 reproduce this error
- Version of WEBKNOSSOS (PyPI): webknossos 0.15.1
- Version of Python: I've tried both 3.9 and 3.10

**Comment by JackieZhai** [comment]
Downgrading `cattrs` from 24.1 to 22.2 `pip install cattrs==22.2` solves this problem.

**Comment by hotzenklotz** [comment]
Fixed by PR https://github.com/scalableminds/webknossos-libs/pull/1184. Released as `webknossos-libs` version [`v0.15.2`](https://github.com/scalableminds/webknossos-libs/releases/tag/v0.15.2).
